common/src/term.rs

```diff
@@ -203,7 +203,7 @@ impl Reporter for SpinnerReporter {
 
     /// Invoked before a new [`Solc`] bin is installed
     fn on_solc_installation_start(&self, version: &Version) {
-        self.send_msg(format!("installing solc version \"{version}\""));
+        self.send_msg(format!("Installing solc version \"{version}\""));
```

Comment:
For consistency with the other messages (no quotes)

```suggestion
        self.send_msg(format!("Installing solc version {version}"));
```